components/client/src/main/java/com/hotels/styx/client/connectionpool/StatsReportingConnectionPool.java

```diff
@@ -91,6 +91,7 @@ private static void registerMetrics(ConnectionPool hostConnectionPool, MetricReg
         scopedRegistry.register("connections-closed", (Gauge<Integer>) () -> (int) stats.closedConnections());
         scopedRegistry.register("connections-terminated", (Gauge<Integer>) () -> (int) stats.terminatedConnections());
         scopedRegistry.register("connections-in-establishment", (Gauge<Integer>) () -> (int) stats.connectionsInEstablishment());
+        scopedRegistry.register("time-to-first-byte-ms", (Gauge<Long>) () -> (long) stats.timeToFirstByteMs());
```

Comment:
I believe CodaHale adds a time unit in the metric itself. Therefore I believe `-ms` suffix is unnecessary. But better to verify this.